.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,12 +13,12 @@ assignees: ''
 2. What did you expect to happen?
 3. What happens instead?
 
-
 **How to reproduce the bug:**
 
 If possible, please include a minimal code snippet here.
 
 **Environment:**
+Go Version: `please paste go version output here`
 ```
 please paste `go env` output here
```

Comment:
Should we also add regular `env` to see `GO11VENDOR` output.

Reply:
I do ask them to paste `go env` underneath